config_and_run/run_in_docker.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+CSV_NAME=$1
+CONFIG_NAME=obscuritext_configs
+
+# Copies csv file and config files into docker
+docker cp $CSV_NAME.csv e37dba6a70c3:/obscuritext-develop/$CSV_NAME.csv
+docker cp $CONFIG_NAME.cfg e37dba6a70c3:/obscuritext-develop/$CONFIG_NAME.cfg
+
+docker exec -it e37dba6a70c3 sh -c "python3 text_obscure.py"
```

Comment:
This hardcoded container ID won't work. The script should take the container ID as an input.